src/test/java/io/openliberty/tools/intellij/it/TestUtils.java

```diff
@@ -541,7 +542,7 @@ public static boolean isServerStopNeeded(String wlpInstallPath) {
      */
     public static void checkDebugPort(String absoluteWLPPath, int debugPort) throws IOException {
         // Retrieve the WLP server.env file path
-        Path serverEnvPath = Paths.get(absoluteWLPPath, "wlp", "usr", "servers", "defaultServer", "server.env");
+        Path serverEnvPath = Paths.get(absoluteWLPPath, "wlp", "usr", "servers", "defaultServer", ItConstants.SERVER_ENV);
```

Comment:
@TrevCraw , Do we need to consider adding all those strings, such as 'wlp', 'usr', 'servers', etc., to the `ITConstants class`? if not then we can keep "server.env" here instead of `ItConstants.SERVER_ENV`